independent-projects/tools/analytics-common/pom.xml

```diff
@@ -61,6 +61,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>uk.org.webcompere</groupId>
+            <artifactId>system-stubs-jupiter</artifactId>
```

Comment:
does this library still work on java 17? expected it would be prevented for module access ?

Reply:
It seems to, It's what I used to build.